src/main/java/core/util/HODateTime.java

```diff
@@ -75,24 +101,50 @@ public String toHT() {
 
     /**
      * Convert to date only string, using user's locale setting (system default)
+     *
      * @return String
      */
     public String toLocaleDate() {
-        var formatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).withZone(ZoneId.systemDefault());
+        return toLocaleDate(FormatStyle.MEDIUM);
+    }
+    public static String toLocaleDate(HODateTime in){
+        if ( in != null) return in.toLocaleDate(FormatStyle.MEDIUM);
+        return "";
+    }
+
+    public String toLocaleDate(FormatStyle style) {
+        var formatter = DateTimeFormatter.ofLocalizedDate(style).withZone(ZoneId.systemDefault());
```

Comment:
This uses the system default, which means it will ignore the custom timezone selected by the user in PReferences, is that intended?

Reply:
Yes, this was intended - do you think, we need to support user preferences too (different time zones for different teams?)

my favorite is to use the system default only.